Marlin/Configuration.h

```diff
@@ -1891,8 +1891,9 @@
 #if ENABLED(NOZZLE_PARK_FEATURE)
   // Specify a park position as { X, Y, Z_raise }
   #define NOZZLE_PARK_POINT { (X_MIN_POS + 10), (Y_MAX_POS - 10), 20 }
-  //#define NOZZLE_PARK_X_ONLY          // X move only is required to park
-  //#define NOZZLE_PARK_Y_ONLY          // Y move only is required to park
+  #define NOZZLE_PARK_MOVE  4			// 0 =  move both X & Y to get to park,
```

Comment:
I'd set this to 0 to maintain current defaults of moving both X & Y at the same time.